scripts/nn/util.dml

```diff
@@ -200,3 +200,19 @@ unpad_image = function(matrix[double] img_padded, int Hin, int Win, int padh, in
   }
 }
 
+threshold = function(matrix[double] X, double thresh)
+    return (matrix[double] out) {
+  /*
+   * Computes an indicator matrix with values in (0, 1) depending on whether or not the values in X
```

Comment:
@dusenberrymw Would it be possible to add some DML recommendations to http://apache.github.io/systemml/contributing-to-systemml.html#dml-code-format or to create a JIRA to update the "DML Code Format" section listing some recommendations such as 72 character line limits, since SystemML really needs more consistency in our DML? The nn library is great in this regard and it would be nice for the rest of our DML to eventually adopt the same great practices.